sections/appendix/stemming.tex

```diff
@@ -1,5 +1,5 @@
 \subsection{Stemming the dataset}\label{sec:stemming}
-In this project, we have done minimal preprocessing of the dataset, because, in a previous project, a more aggressive preprocessing step turned out to hurt the performance of the \gls{lda} model.
+In this project, we have done minimal preprocessing \vejleder{hvad med ?? ord (se dokument)}of the dataset, because, in a previous project, a more aggressive preprocessing step turned out to hurt the performance of the \gls{lda} model.
```

Comment:
```suggestion
\vejleder{Se dokument for flere kommentarer}
In this project, we have done minimal preprocessing \vejleder{hvad med stop ord, fylde ord?}of the dataset, because, in a previous project, a more aggressive preprocessing step turned out to hurt the performance of the \gls{lda} model.
```
Tror jeg han skriver